README.md

```diff
@@ -1,3 +1,67 @@
+<div align="center" width="100px">
+
+ <picture>
+   <source media="(prefers-color-scheme: dark)" srcset="https://github.com/user-attachments/assets/76e3079f-696a-4fcd-8658-89739647090b">
+   <source media="(prefers-color-scheme: light)" srcset="https://github.com/user-attachments/assets/8477d643-a905-4c63-8ed3-03d0976f6fc3">
+   <img width="200" alt="saleor-commerce-logo" src="https://user-images.githubusercontent.com/4006792/214636328-8e4f83e8-66cb-4114-a3d8-473eb908b9c3.png">
+
+ </picture>
+</div>
+
+<div align="center">
+  <strong>Commerce that works with your language and stack</strong>
+</div>
+
+<div align="center">
+  GraphQL native, API-only platform for scalable composable commerce.
+</div>
+
+<br>
+
+<div align="center">
+ Get to know Saleor: <br>
+  <a href="https://cloud.saleor.io/signup?utm_source=github&utm_medium=readme&utm_campaign=repo_saleor-docs">Talk to a human</a>
+  <span> | </span>
+  <a href="https://saleor.typeform.com/talk-with-us?utm_source=github&utm_medium=readme&utm_campaign=repo_saleor-docs">Talk to API</a>
```

Comment:
Aren't the links inverted? "Talk to a human" links to Cloud sign-up page, and "Talk to API" links to contact form